src/features/contact/contactverzoek/contactverzoekLogboek/LogboekOverzicht.vue

```diff
@@ -287,7 +287,10 @@ const getActionTitle = (type: string) =>
   activiteitTitles.get(type) || "Onbekende actie";
 
 async function fetchObject(id: string) {
-  return await fetchLoggedIn("/api/afdelingen/api/v2/objects/" + id)
+  return await fetchLoggedIn(
+    "api/v2/objects?ordering=record__data__identificatie&data_attr=identificatie__exact__" +
```

Comment:
Dit lijkt me op zich de juiste oplossing, maar waar dient de ordering parameter voor in deze situatie?
En is de response niet anders tov de oude code? daar werd specifiek 1 item opgevraagd, met deze nieuwe call verwacht ik dat je een lijst terugkrijgt.

Reply:
svp de pr naam voortaan gelijkhouden aan de branchnaam.

Reply:
heb je dit getest? volgens mij kan dit niet gewerkt hebben. Ik heb wat gerefactored. svp reviewen

Reply:
besproken